horace_core/utilities/split_data_blocks.m

```diff
@@ -1,16 +1,18 @@
-function [chunks, cumulative_sum] = split_data_blocks(start_val,block_sizes, max_chunk_sum)
+function [chunks, cumulative_sum] = split_data_blocks(start_pos,block_sizes, buf_size)
 % Split the array of the data blocks into the blocks, which have equal or
 % almost equal size
 %
 % Input:
 % ------
-% start_val          A vector of numeric, non-negative, values describing a
-%                    position of a block on some media
+% start_pos          A vector of numeric, non-negative, values describing a
```

Comment:
the positions

Reply:
that the number of variable so described as it is